tests/Mvc/Database/DataImporterUnitTest.php

```
@@ -0,0 +1,656 @@
+
+/**
+ * Unit tests for DataImporter with mocked dependencies
+ */
+class DataImporterUnitTest extends TestCase
+{
+	/**
+	 * Original AdapterFactory instance to restore after tests
+	 *
+	 * @var mixed
+	 */
+	private $originalFactory;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$originalFactory'. (undefined)

(UnusedPrivateField)